### PR TITLE
Adjust quite a bit of wording

### PR DESCRIPTION
**Pull request by wafflecoffee**
Also converts many title case things to sentence case, and adds a link on the edit mod & create mod pages to the licensing guide

### CODE REVIEW

File: pages/mod/create.vue [hunk 1]
```diff
@@ -364,8 +360,8 @@
             <h3>Changelog</h3>
             <span>
               Tell players and modpack makers what's new. It supports the same
-              markdown as description, but it is advisable not to be too
-              creative with it in changelogs
+              Markdown as the description, but it is advised not to be too
```
Comment (WickedTree) [inline pages/mod/create.vue:363]: Maybe changed `advised` to `recommended`?

**Review by WickedTree — APPROVED** [review]
Just 1 small possible change. Looks good :D

**Review by WickedTree — APPROVED** [review]

**Review by Redblueflame — APPROVED** [review]

**Review by venashial — APPROVED** [review]

**Comment by Minenash** [comment]
Is there a reason modrinth is using sentence cases in places that normally are title case, like actual titles? Personally it feels off and "unpolished"

**Comment by wafflecoffee** [comment]
Per https://falseresync.ru/blog/modrinth-knossos-design-guidelines/:
> * Non-initial words SHOULD NOT be capitalized, e.g. "Login with my account", not "Login With My Account"

**Comment by Minenash** [comment]
Well yes, but that doc isn't stone. But, I was more wondering why that guideline was in the doc. After seeing it used, it looks off, and imo the sections that were entirely capital case did look better

**Review by Geometrically — APPROVED** [review]

**Pull request merged**